spring-examples/springboot-clustering/springboot-clustering-hazelcast/src/main/java/io/vertx/examples/spring/clustering/hazelcast/VertxProducer.java

```diff
@@ -47,8 +47,11 @@
 
   @PostConstruct
   void init() throws ExecutionException, InterruptedException {
-    VertxOptions options = new VertxOptions()
-      .setClusterManager(new HazelcastClusterManager(hazelcastInstance));
+    		VertxOptions options = new VertxOptions()
+			      .setClusterManager(new HazelcastClusterManager(hazelcastInstance))
+			      .setClustered(true)
+			      .setHAEnabled(true)
+			      .setClusterHost("192.168.0.67");// Write your ip address numeric form
```

Comment:
You can't do this otherwise no one can run the examples without modifying source. You could introduce a Spring config property for cluster host. If not set, `setClusterHost` would not be called